runtime/kusama/src/xcm_config.rs

```diff
@@ -90,7 +91,7 @@ type LocalOriginConverter = (
 
 parameter_types! {
 	/// The amount of weight an XCM operation takes. This is a safe overestimate.
-	pub const BaseXcmWeight: u64 = 1_000_000_000;
+	pub const BaseXcmWeight: Weight = Weight::from_ref_time(1_000_000_000);
```

Comment:
should be `from_parts` and use a sensible upper limit for the amount of proof size an XCM instruction could reasonably take.